### PR TITLE
A sea more sunless 

### PR DESCRIPTION
**Pull request by drexample**
## About The Pull Request

New map, y'all! New layout [same great taste]!

## Why It's Good For The Game

Fresher and more fitting to our needs!

## Changelog
:cl:
add: New map!
/:cl:

### CODE REVIEW

File: tools/hooks/Install.bat [hunk 1]
```diff
@@ -1,2 +0,0 @@
-@call "%~dp0\..\bootstrap\python" -m hooks.install %*
-@pause
```
Comment (EvilJackCarver) [inline tools/hooks/Install.bat:2]: Should probably leave this file in.

**Review by EvilJackCarver — CHANGES_REQUESTED** [review]

File: tools/hooks/Install.bat (hunk 1 above)
Comment (drexample) [reply tools/hooks/Install.bat:2]: image
bruh it auto-ignores it even when I manually add it, what is this witchcraft

Comment (EvilJackCarver) [reply tools/hooks/Install.bat:2]: wgat

nestor what have you DONE

Comment (drexample) [reply tools/hooks/Install.bat:2]: Adding a newline fixed it, but I have no idea what broke it. I didn't *touch* the file.

**Comment by EvilJackCarver** [comment]
Cleared for merge once checks finish.

**Pull request merged**